plugins/fireblocks/provider-gas-wrapper.ts

```diff
@@ -15,29 +14,29 @@ import { BigNumber } from 'ethers';
 // curl -H "Authorization: xxxx" https://api.blocknative.com/gasprices/blockprices
 // {"system":"ethereum","network":"main","unit":"gwei","maxPrice":55,"currentBlockNumber":14562255,"msSinceLastBlock":29419,"blockPrices":[{"blockNumber":14562256,"estimatedTransactionCount":431,"baseFee":31.171002417,"estimatedPrices":[{"confidence":99,"price":33,"maxPriorityFee":2,"maxFee":64.34},{"confidence":95,"price":32,"maxPriorityFee":1.62,"maxFee":63.96},{"confidence":90,"price":32,"maxPriorityFee":1.51,"maxFee":63.85},{"confidence":80,"price":32,"maxPriorityFee":1.5,"maxFee":63.84},{"confidence":70,"price":32,"maxPriorityFee":1.17,"maxFee":63.51}]}]}%
 
-// Sensible defaults -- TODO query the gas station services above
+// Sensible defaults -- all in gwei
 const defaultGasFeeSettings: {
-  [key: number]: Pick<FeeData, 'maxFeePerGas' | 'maxPriorityFeePerGas'>;
+  [key: number]: { maxFeePerGas: string; maxPriorityFeePerGas: string };
 } = {
   137: {
-    maxFeePerGas: BigNumber.from(120),
-    maxPriorityFeePerGas: BigNumber.from(60),
+    maxFeePerGas: '120',
+    maxPriorityFeePerGas: '60',
   },
   80_001: {
-    maxFeePerGas: BigNumber.from(30),
-    maxPriorityFeePerGas: BigNumber.from(30),
+    maxFeePerGas: '30',
+    maxPriorityFeePerGas: '30',
   },
   1: {
-    maxFeePerGas: BigNumber.from(80),
-    maxPriorityFeePerGas: BigNumber.from(2),
+    maxFeePerGas: '80',
+    maxPriorityFeePerGas: '2',
   },
   5: {
-    maxFeePerGas: BigNumber.from(70),
-    maxPriorityFeePerGas: BigNumber.from(2),
+    maxFeePerGas: '70',
+    maxPriorityFeePerGas: '2',
   },
 };
 
-type GAS_SPEED = 'safeLow' | 'standard' | 'fast';
+type GasSpeed = 'safeLow' | 'standard' | 'fast';
```

Comment:
nit: we could use this like a true enum, and avoid references to these strings elsewhere
```suggestion
enum GasSpeed {
  SAFE_LOW = 'safeLow',
  STANDARD = 'standard',
  FAST = 'fast',
}
```

Reply:
I'll give that a try.  This was lazily using the naming that comes in from polygon so this would require more parsing there but a good idea.